modules/images-configuration-insecure.adoc

```diff
@@ -35,6 +35,8 @@ metadata:
   selfLink: /apis/config.openshift.io/v1/images/cluster
   uid: e34555da-78a9-11e9-b92b-06d6c7da38dc
 spec:
+  allowedRegistries:
```

Comment:
**allowedRegistries** should under spec.registrySources, and the vlaues should be in a list without domainName keyword.
e.g. 
spec:
  registrySources:
    allowedRegistries:
    - quay.io
See the definition https://github.com/openshift/api/blob/master/config/v1/types_image.go#L101-L114

Reply:
Should add explain for allowedRegistries too.

Reply:
Thank you! Updated.